.travis.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - stage: testing
       name: "Run unit tests"
       before_script:
-        - pip3 install pyflakes pycodestyle
+        - pip3 install pyflakes pycodestyle pytest
```

Comment:
As per https://github.com/cobbler/cobbler/blob/a881ab482d21354510e6e1d515b42e4ead897746/setup.py#L546 this is not needed.

Reply:
Travis seems to be somehow broken. Every PR is currently failing...